CHANGELOG.md

```diff
@@ -1,3 +1,35 @@
+# dbt_snapchat_ads_source v0.8.0
+[PR #27](https://github.com/fivetran/dbt_snapchat_ads_source/pull/27) includes the following updates:
+
+## Schema Updates
+
+**5 total changes • 0 breaking changes**
+| **Data Model** | **Change type** | **Old name** | **New name** | **Notes** |
+| ---------------- | --------------- | ------------ | ------------ | --------- |
+| `stg_snapchat_ads__campaign_geo_country_daily_report` | New Staging Model |   |   |  Uses `campaign_geo_country_daily_report` source table  |
+| `stg_snapchat_ads__campaign_geo_region_daily_report` | New Staging Model |   |   |  Uses `campaign_geo_region_daily_report` source table  |
+| `stg_snapchat_ads__campaign_geo_country_daily_report_tmp` | New Temp Model |   |   |  Uses `campaign_geo_country_daily_report` source table  |
+| `stg_snapchat_ads__campaign_geo_region_daily_report_tmp` | New Temp Model |   |   |  Uses `campaign_geo_region_daily_report` source table  |
+| `stg_snapchat_ads__campaign_history` | New Columns |   |  `daily_budget_micro`, `start_time`, `end_time`, `lifetime_spend_cap_micro`, `status`, `objective  |    |
```

Comment:
`objective needs to have closed quotes (`objective`)

Reply:
oh thanks for catching!